python/tvm/relay/frontend/tflite.py

```diff
@@ -743,6 +744,31 @@ def convert_split(self, op):
 
         return out
 
+    def convert_transpose(self, op):
+        """transpose implementation."""
+        try:
+            from tflite.Operator import Operator
+        except ImportError:
+            raise ImportError("The tflite package must be installed")
+
+        assert isinstance(op, Operator)
+        input_tensors = self.get_input_tensors(op)
+        assert len(input_tensors) == 2, "input tensors length should be 2"
+        input_tensor = input_tensors[0]
+        input_tensor_idx = input_tensor.tensor_idx
+
+        in_expr = self.get_expr(input_tensor_idx)
+
+        # axis
+        in_axis = tuple(self.get_tensor_value(input_tensors[1]))
+
+        if in_axis is None:
```

Comment:
Could we have the condition `in_axis` be None? I think we will never meet this condition. Because we construct it using tuple constructor. The bad case should be `()` in my opinion. Could you check the code here again and write one unit test to cover?

Reply:
Thanks for reviewing. I improve the condition and add one unit test to cover empty tuple.